index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <main class="main-container">
+      <header class="toolbar">
+        <h1>Welcome</h1>
```

Comment:
- [ ] Kindly make this an `a` tag. Normally, logos are meant to work as links to the homepage.